prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -528,16 +544,31 @@ class PutMultipleSettings : SettingCommand(
         deepOrgs.forEach { deepOrg ->
             val org = Organization(deepOrg)
             val payload = jsonMapper.writeValueAsString(org)
+
```

Comment:
How does the output look with this being in a loop? Just curious of the readability if there are multiple organizations, senders and receivers.

Reply:
![image](https://user-images.githubusercontent.com/71846466/121903800-09519380-ccff-11eb-96e5-f69268acf0e4.png)